<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="aliceGB">
     <property name="title">
      <string>Alice</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="pAliceLabel">
        <property name="text">
         <string>p</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="pAliceCB"/>
      </item>
      <item row="0" column="2" colspan="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>159</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="qAliceLabel">
        <property name="text">
         <string>q</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="qAliceCB"/>
      </item>
      <item row="1" column="2" colspan="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>159</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="nAliceLabel">
        <property name="text">
         <string>N</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="nAliceValueLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>159</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="kAliceLabel">
        <property name="text">
         <string>K</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="kAliceValueLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>159</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="modeAliceLabel">
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="modeAliceCB">
        <item>
         <property name="text">
          <string>Numbers</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Text</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <spacer name="horizontalSpacer_15">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>160</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="QListWidget" name="aliceLW">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="inputAliceLabel">
        <property name="text">
         <string>Input</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="4">
       <widget class="QTextEdit" name="inputAliceTE">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>150</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="outputAliceLabel">
        <property name="text">
         <string>Output</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="4">
       <widget class="QTextEdit" name="outputAliceTE">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>150</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>38</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="1">
       <widget class="QPushButton" name="codeAndSendAlicePB">
        <property name="text">
         <string>Code and Send</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QPushButton" name="encodeAlicePB">
        <property name="text">
         <string>Encode</string>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <spacer name="horizontalSpacer_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>21</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="bobGB">
     <property name="title">
      <string>Bob</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="pBobLabel">
        <property name="text">
         <string>p</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="pBobCB"/>
      </item>
      <item row="0" column="2" colspan="2">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>157</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="qBobLabel">
        <property name="text">
         <string>q</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="qBobCB"/>
      </item>
      <item row="1" column="2" colspan="2">
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>157</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="nBobLabel">
        <property name="text">
         <string>N</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="nBobValueLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>157</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="kBobLabel">
        <property name="text">
         <string>K</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="kBobValueLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>157</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="modeBobLabel">
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="modeBobCB">
        <item>
         <property name="text">
          <string>Numbers</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Text</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="QListWidget" name="bobLW">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="inputBobLabel">
        <property name="text">
         <string>Input</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="4">
       <widget class="QTextEdit" name="inputBobTE">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>150</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="outputBobLabel">
        <property name="text">
         <string>Output</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="4">
       <widget class="QTextEdit" name="outputBobTE">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>150</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <spacer name="horizontalSpacer_13">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>38</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="1">
       <widget class="QPushButton" name="codeAndSendBobPB">
        <property name="text">
         <string>Code and Send</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QPushButton" name="encodeBobPB">
        <property name="text">
         <string>Encode</string>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <spacer name="horizontalSpacer_14">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>21</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="2" colspan="2">
       <spacer name="horizontalSpacer_16">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>281</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>341</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>732</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>725</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
